index.d.ts

```diff
@@ -0,0 +1,664 @@
+
+import { EventEmitter } from 'events'
+import puppeteer = require('puppeteer')
+
+declare namespace WhatsappWeb {
```

Comment:
```suggestion
declare namespace WAWebJS {
```

Reply:
Ok, no problem. This might be a better name. 
In the end there would be no practical difference, because we can import the lib (and the types) with a custom name. E.g.:

```typescript
import WhatsappWebCustom = require('whatsapp-web.js')
import { Client as CustomClient, MessageId } from 'whatsapp-web.js'
```

Reply:
Wait, I forgot we need to also rename all the WhatsappWeb.* entries in the file.